### PR TITLE
Small change

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Comment by cclauss** [comment]
```
Error: stock_app.py:28:24: FBT003 Boolean positional value in function call
Error: stock_app.py:28:31: FBT003 Boolean positional value in function call
Error: stock_app.py:83:16: BLE001 Do not catch blind exception: `Exception`
Error: stock_app.py:84:67: RUF010 Use explicit conversion flag
Error: stock_app.py:91:9: PD901 `df` is a bad variable name. Be kinder to your future self.
Error: stock_app.py:92:9: PD901 `df` is a bad variable name. Be kinder to your future self.
Error: stock_app.py:100:18: FBT003 Boolean positional value in function call
Error: stock_app.py:104:9: PD901 `df` is a bad variable name. Be kinder to your future self.
Error: stock_app.py:105:9: PD901 `df` is a bad variable name. Be kinder to your future self.
Error: stock_app.py:123:13: PERF203 `try`-`except` within a loop incurs performance overhead
Error: stock_app.py:123:20: BLE001 Do not catch blind exception: `Exception`
Error: stock_app.py:124:84: RUF010 Use explicit conversion flag
Error: stock_app.py:130:18: FBT003 Boolean positional value in function call
Error: stock_app.py:142:9: PD901 `df` is a bad variable name. Be kinder to your future self.
Error: stock_app.py:143:9: PD901 `df` is a bad variable name. Be kinder to your future self.
Error: stock_app.py:158:18: FBT003 Boolean positional value in function call
Error: stock_app.py:170:9: SIM108 Use ternary operator `portfolio_info = "\n".join(self.portfolio) if self.portfolio else "Portfolio is empty."` instead of `if`-`else`-block
Error: stock_app.py:176:1: W293 Blank line contains whitespace
```
% `ruff rule FBT003`
# boolean-positional-value-in-call (FBT003)

Derived from the **flake8-boolean-trap** linter.

## What it does
Checks for boolean positional arguments in function calls.

## Why is this bad?
Calling a function with boolean positional arguments is confusing as the
meaning of the boolean value is not clear to the caller, and to future
readers of the code.

## Example
```python
def func(flag: bool) -> None:
    ...


func(True)
```

Use instead:
```python
def func(flag: bool) -> None:
    ...


func(flag=True)
```

## References
- [Python documentation: Calls](https://docs.python.org/3/reference/expressions.html#calls)
- [_How to Avoid “The Boolean Trap”_ by Adam Johnson](https://adamj.eu/tech/2021/07/10/python-type-hints-how-to-avoid-the-boolean-trap/)

**Comment by debnath003** [comment]
I am trying to change the code base on the current faults

File: stock_app.py [hunk 1]
```diff
@@ -101,10 +111,12 @@ def analyze_and_plot(self, symbol, data):
...
     def plot_stock_data(self, symbol, data):
-        df = data.reset_index()[['Date', 'Close']]
-        df = df.rename(columns={'Date': 'ds', 'Close': 'y'})
+        df = data.reset_index()[["Date", "Close"]]
+        df = df.rename(columns={"Date": "ds", "Close": "y"})
 
-        plt.plot(df['ds'], df['y'], label=symbol.replace(" ", "_"))  # Use a suitable label format
+        plt.plot(
+            df["ds"], df["y"], label=symbol.replace(" ", "_")
+        )  # Use a suitable label format
```
Comment (cclauss) [inline stock_app.py]: Don't wrap lines to fit a comment.
```suggestion
        # Use a suitable label format
        plt.plot(df["ds"], df["y"], label=symbol.replace(" ", "_"))
```

File: stock_app.py [hunk 2]
```diff
@@ -80,19 +90,19 @@ def analyze_stock(self):
...
             return
 
     def fetch_stock_data(self, symbol, start_date, end_date):
         return yf.download(symbol, start=start_date, end=end_date)
 
     def analyze_and_plot(self, symbol, data):
-        df = data.reset_index()[['Date', 'Close']]
-        df = df.rename(columns={'Date': 'ds', 'Close': 'y'})
+        df = data.reset_index()[["Date", "Close"]]
+        df = df.rename(columns={"Date": "ds", "Close": "y"})
```
Comment (cclauss) [inline stock_app.py]:
```suggestion
        date_close = data.reset_index()[["Date", "Close"]]
        date_close = date_close.rename(columns={"Date": "ds", "Close": "y"})
```
Repeat below.  `date_close_df` or `date_close_dataframe` would also be acceptable.

File: stock_app.py [hunk 3]
```diff
@@ -25,23 +25,29 @@ def __init__(self):
...
 
         self.symbol_label = tk.Label(self, text="Enter A Stock Symbol:")
         self.symbol_label.pack()
 
         self.symbol_entry = tk.Entry(self)
         self.symbol_entry.pack()
 
-        self.add_to_portfolio_button = tk.Button(self, text="Add to Portfolio", command=self.add_to_portfolio)
+        self.add_to_portfolio_button = tk.Button(
+            self, text="Add to Portfolio", command=self.add_to_portfolio
```
Comment (cclauss) [inline stock_app.py:58]: I always `--ignore=COM`

**Comment by debnath003** [comment]
Can you please edit the code? Like what are the errors associated with it and how to tackle it @cclauss ?

File: stock_app.py [hunk 4]
```diff
@@ -9,7 +9,6 @@
 from prophet import Prophet
 from tkcalendar import DateEntry
 
-
```
Comment (cclauss) [inline stock_app.py:10]: Black will autofix this for you...  Two blank lines before a `class`

Comment (debnath003) [reply stock_app.py:10]: Yeah I have done that 😁

**Comment by debnath003** [comment]
See I tried @cclauss but this formatting error and docstring error are very hectic tbh 🫠So I am closing the PR for you, I still have made some important commits; if you really want to check it out and format the code, give it a try! I hope it will go successfully! Cheers ;)

**Pull request merged**